src/App.tsx

```diff
@@ -86,9 +87,14 @@ const App = () => {
 
   useMount(() => {
     dispatch(restoreAppSettings())
+  })
+
+  useEffect(() => {
+    if (!settingsLoaded) return
```

Comment:
Does fetchAccountSettings only get dispatched on the more screen? Should we also call it here to makes sure it loads and doesnt block the app?

Reply:
It gets called in App.tsx by the `useAccountSettings` hook as well.